test/entry/defaults-index/entry-multi-args.test.js

```diff
@@ -10,8 +10,9 @@ describe('single entry flag index present', () => {
         const { stdout, stderr } = run(__dirname);
         const summary = extractSummary(stdout);
         const outputDir = 'entry/defaults-index/bin';
-
-        expect(summary['Output Directory']).toContain(outputDir);
+        const outDirectoryFromCompiler = summary['Output Directory'].split(sep);
+        const outDirToMatch = outDirectoryFromCompiler.slice(outDirectoryFromCompiler.length - 3, outDirectoryFromCompiler.length).join('/');
```

Comment:
need a util for these

Reply:
test utils ? like extracting these lines ?

Reply:
yup, make them a utility function in `test-utils`, use that function instead for all occurences